cfme/tests/control/test_actions.py

```diff
@@ -150,7 +145,8 @@ def vm(request, provider, small_template, vm_name):
             pytest.skip("{} is quite likely overloaded! Check its status!\n{}: {}".format(
                 provider.key, type(e).__name__, str(e)))
 
-    def finalize():
+    @request.addfinalizer
+    def _finalize():
```

Comment:
@seandst It should be ok to merge, there are some issues with rhos but I want to resolve that when the provider dry comes in. This should kill off the vsphere6 errors